frontend/src/theme/utils.ts

```diff
@@ -297,6 +298,7 @@ const computeDerivedColors = (
 
   const fadedText05 = transparentize(bodyText, 0.9) // Mostly used for very faint 1px lines.
   const fadedText10 = transparentize(bodyText, 0.8) // Mostly used for 1px lines.
+  const fadedText20 = transparentize(bodyText, 0.7) // Used for 1px lines.
   const fadedText40 = transparentize(bodyText, 0.6) // Backgrounds.
   const fadedText60 = transparentize(bodyText, 0.4) // Secondary text.
```

Comment:
Hm, probably something to fix in a followup PR (and also not very high priority), but shouldn't these names (from the top down) be

`fadedText10`, `fadedText20`, `fadedText30`, etc?

Reply:
That would make substantially more sense to me at least, adding to my cleanup list 👍🏼